package.json

```diff
@@ -75,7 +75,7 @@
   },
   "devDependencies": {
     "@google-cloud/bigquery": "^4.0.0",
-    "@google-cloud/nodejs-repo-tools": "^3.0.0",
+    "@google-cloud/nodejs-repo-tools": "^3.3.0",
```

Comment:
there's an open PR for the repo-metadata.json, at which point we can just get rid of this dependency all together.